resources/helm/v2.1/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -168,29 +160,9 @@ spec:
             value: "{{ $.Values.global.multiCluster.clusterName | default `Kubernetes` }}"
           - name: EXTERNAL_ISTIOD
             value: "{{ $.Values.global.externalIstiod | default "false" }}"
-          - name: CENTRAL_ISTIOD
-            value: "{{ $.Values.global.centralIstiod | default "false" }}"
-{{- if .Values.global.tls }}
-{{- if .Values.global.tls.minProtocolVersion }}
```

Comment:
hmm.. this stuff is part of the TLS customization changeset we have in maistra. it shouldn't get lost

Reply:
It's back now after the latest rebase.